tsdb_k8s_queries/container-template.json

```diff
@@ -11,9 +11,6 @@
           {# non-serverless-index-settings-marker-start #}{%- if build_flavor != "serverless" or serverless_operator == true -%}
```

Comment:
`!= "serverless"`  should not this be == "serverless"  ?

Also why we need to variables and not only one?